packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -946,28 +946,28 @@ Expected response body:
 }
 ```
 
-### Azure_ResourceManager_Models_Resources_NestedProxyResources_delete
+### Azure_ResourceManager_Resource_Nested_delete
```

Comment:
is the fact that it's a proxy resource part of the test? If so, we can definitely include `NestedProxyResource`. I do think resources should still be plural for grammatic sense. So either `Azure_ResourceManager_Resources_NestedProxyResource_delete` (note the first Resources is plural, but the second one is singular. It's because `Resources` is a grouping of all of the resources, while proxy resource is a singular resource in the list of `Resources`). Or it could be `Azure_ResourceManager_Resources_Nested_delete` if proxy doesn't matter to the test